baseband/vdif/header.py

```diff
@@ -28,6 +32,41 @@
                   precision=9)
 
 
+_vdif_edv_header_classes = {}
+"""Dict for storing pairs of VDIF EDV and header classes."""
+
+class _VDIFHeaderRegistry(type):
+    """
+    Registry of VDIF Header EDV types, using the ``_vdif_edv_header_classes``
+    dict.  Checks for keyword and subclass conflicts before registering.
+    """
+    _vdif_edv_header_classes = _vdif_edv_header_classes
+
+    def __init__(cls, name, bases, dct):
+
+        # Check that class is subclass of VDIFHeader
+        if not issubclass(cls, VDIFHeader):
```

Comment:
This cannot happen - the meta class will only get called on a subclass of `VDIFHeader` -- see comment below as well. So, this can be removed.